Makefile

```diff
@@ -403,11 +403,12 @@ _build/NoInit.proxy: _build/Base.proxy
 	touch $@
 
 _build/PoplogCommander.proxy: _build/Stage2.proxy
-	mkdir -p _build/cmdr
-	GETPOPLOG_VERSION="$(GETPOPLOG_VERSION)" sh makePoplogCommander.sh > _build/cmdr/poplog.c
-	( cd _build/cmdr && $(CC) $(CFLAGS) -Wextra -Werror -Wpedantic -o poplog poplog.c )
+	mkdir -p _build/commander
+	mkdir -p _build/poplog_base/pop/bin
+	( GETPOPLOG_VERSION="$(GETPOPLOG_VERSION)" bash makePoplogCommander.sh ) > _build/commander/poplog.c
+	( cd _build/commander && $(CC) $(CFLAGS) -Wextra -Werror -Wpedantic -o poplog poplog.c )
 	rm -f _build/poplog_base/pop/pop/poplog
```

Comment:
Shouldn't this be `pop/bin/poplog`? Or removed (what's the point of it?)

Reply:
Fixed.